proxy/manifest.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "Proxy Boilerplate Code",
+  "manifest_version": 2,
+  "version": "1.0",
+  "background": {
+    "scripts": ["background.js"]
+  },
+  "permissions": [
+    "proxy"
+  ],
+  "description": "Provides boilerplate code to get started with the Proxy API.",
```

Comment:
The proxy script doesn't have access to console.log, but it can send messages to the background script which could log the message it receives. I'll add an example of this.